api/services/connect/schemas/EntityBlock.yaml

```
@@ -1,13 +1,18 @@
 properties:
   id:
+    description: Spot id of the user
```

Use `ID` instead of `id` for consistency with the current convention.
```suggestion
    description: Spot ID of the user
```